content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autocomponents.md

```diff
@@ -4,17 +4,18 @@ description: A reference for defining Open 3D Engine multiplayer state through a
 linktitle: Auto-components
 ---
 
-*Auto-components* provide a convenient way to define states of a multiplayer component relevant to network synchronization. Using the [AzAutoGen](/docs/user-guide/programming/autogen) system, auto-component files found inside of your project are processed during builds to create C++ classes for components and controllers that provide network replication and remote function calls. Auto-components also take care of Editor and behavior context bindings so that the bound component shows in the Editor and works with O3DE scripting.
+*Auto-components* provide a convenient way to define states of a multiplayer component that's relevant to network synchronization. Using the [AzAutoGen](/docs/user-guide/programming/autogen) system, auto-component files in your project are processed at build-time to automatically generate C++ classes for components and controllers that provide network replication and remote function calls. Auto-components also take care of [edit](/docs/user-guide/programming/components/reflection/edit-context) and [behavior](/docs/user-guide/programming/components/reflection/behavior-context) context bindings, so that the bound component shows in the Editor and works with O3DE scripting.
 
-In order to enable auto-component builds for your project, follow the instructions in [Multiplayer Project Configuration](./configuration).
+To enable auto-component builds for your project, follow the instructions in [Multiplayer Project Configuration](./configuration).
```

Comment:
It's also possible to use Project Manager's (o3de.exe) remote projects to start a multiplayer game. 
Is there already a docs page about remote templates we could point to? The remote url that contains the multiplayer template is at https://github.com/o3de/o3de-extras.git (see screenshot)
This is much faster than enabling auto-components by hand.
![image](https://user-images.githubusercontent.com/32776221/199835755-4cd817b1-5508-48dd-8f96-304a5a55711d.png)

Reply:
Yes! There's an in-progress issue about that: https://github.com/o3de/o3de.org/issues/2052
The updates will be for the configuration page, so the link here as it is will direct users there.